whois-commons/src/main/java/net/ripe/db/whois/common/support/TelnetWhoisClient.java

```diff
@@ -93,7 +93,7 @@ public String sendQuery(final String query, final Charset charset, final int tim
         try {
             return sendQueryWithRetry(query, function, charset, timeoutMs);
         } catch (IOException e) {
-            final String message = String.format("Error querying for '%s' at '%s': %s", query, host, e.getMessage());
+            final String message = String.format("Error querying for '%s' at '%s':'%s' %s", query, host, port, e.getMessage());
```

Comment:
this.port is an int, use %d in String.format (and no need for the single quotes around the port number maybe?)